stable/openvpn/templates/certs-pvc.yaml

```diff
@@ -4,6 +4,7 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
We use namespaces + calico firewall to great effect: a vpn is something we very much need to be namespaced.